core/utils/password.go

```diff
@@ -25,27 +25,41 @@ var LeadingWhitespace = regexp.MustCompile(`^\s+`)
 var TrailingWhitespace = regexp.MustCompile(`\s+$`)
 
 var (
-	ErrPasswordMinLength = errors.Errorf("must be longer than %d characters", MinRequiredLen)
-	ErrWhitespace        = errors.New("must not contain leading or trailing whitespace characters")
+	ErrMsgHeader = fmt.Sprintf(`
+Expected password complexity:
+Must be longer than %d characters
+Must not comprise:
+	Leading or trailing whitespace
+	A user's API email
+
+Faults:
+`, MinRequiredLen)
+	ErrWhitespace = errors.New("Password contains a leading or trailing whitespace")
```

Comment:
AFAIK errors shall not start with capital case letter (https://github.com/golang/go/wiki/CodeReviewComments#error-strings)
but maybe you don't need this variable to be an error.